cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -30,6 +30,7 @@ function($, Backbone, _, gettext, BaseView, XBlockViewUtils, MoveXBlockUtils, Ht
         options: $.extend({}, BaseModal.prototype.options, {
             modalName: 'move-xblock',
             modalSize: 'lg',
+            modalTop: 110,
```

Comment:
Edit modal has fix height where as move modal changes height dynamically. We are adding this into move modal only because only move modal is having issue.